data/json/items/ammo.json

```diff
@@ -422,7 +422,7 @@
         "volume" : 1,
         "weight" : 11,
         "ammo_type" : "12mm",
-        "damage" : 45,
```

Comment:
I thought this was left intentional in ammo.json to avoid a file for just one item.